src/context/NotificationContext.tsx

```diff
@@ -1,12 +1,19 @@
-import React, { useState, useCallback } from 'react';
+import React, { useState, useCallback, useMemo } from 'react';
 import Notification, { INotificationProps } from '../Alerts/atom/Notification';
 
+
+const uniqueID = (): string => {
+  return `${Date.now()}`;
```

Comment:
when many notices happen in a similar moment Date.now() can have collisions.  
![image](https://user-images.githubusercontent.com/779444/179901386-6f71830a-a185-4813-bd73-702d5ef25a8d.png)
 See here:
 
![image](https://user-images.githubusercontent.com/779444/179901502-4263298f-479c-4101-9c46-7698f9f85aff.png)


Alternatively we could just add some random number to date.now to get less collisions.  

https://developer.mozilla.org/en-US/docs/Web/API/Performance/now

Reply:
Maybe this is just over engineering tho.

Reply:
I did gave a though on this but at the end I thought that maybe having the date will be useful in the future.

This article mentioned that if your "Trigger is user click" the date should be enough but also suggested some interesting ideas

https://dev.to/rahmanfadhil/how-to-generate-unique-id-in-javascript-1b13

I'll use this suggestion


```
const uniqueID = () =>
  String(
    Date.now().toString(32) +
      Math.random().toString(16)
  ).replace(/\./g, '');
  ```

Reply:
This works, the issue is if 2 notifications are triggered by a user click at the same time.